content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -81,6 +81,64 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: A string representing an application environment name (for example, `prod`, `dev`) <br>
 **Description**: Adds an environment tag to all spans generated by the tracer instance.
 
+`DD_TAGS`
+: **Default**: `null` <br>
+**Supported Input**: A string representing key value pairs that are delimited by a colon and separated by a comma and/or a space (for example, `<key1>:<value1>, <key2>:<value2>`>) <br>
+**Description**: Tags to apply to produced data. Must be a list of `<key>:<value>` separated by commas and/or spaces.
+
+`DD_TRACE_SAMPLING_RULES`
+: **Default**: `null` <br>
+**Supported Input**: A JSON string representing an array of sampling rules. Each rule must contain a `sample_rate` (float between 0 and 1) and can optionally include a `service` (string) and `name` (string) pattern to match. Example: `[{"sample_rate": 0.1, "service": "my-service", "name": "http.request"}]` <br>
+**Description**: Configures custom sampling rules for traces. Rules are evaluated in order, and the first matching rule determines the sampling rate. If no rules match, the default sampling rate is used.
+
+`DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean (`true`/`false`) <br>
+**Description**: Enables or disables the generation of 128-bit trace IDs. When enabled, trace IDs are generated as 32-character hexadecimal strings. When disabled, 64-bit trace IDs are used instead.
```

Comment:
For both of the 128-bit toggles, I don't think it's immediately clear _why_ someone might want to set this. Can you add a description? Off the top of my head, I think something like `The latest Datadog tracing libraries support W3C trace IDs (128-bit) and generate new traces with W3C trace IDs by default. To generate trace IDs in the previous Datadog (64-bit) format, you can set this configuration to false.`